sample/android/src/main/res/layout/item_button.xml

```diff
@@ -8,18 +8,13 @@
         <variable
             name="viewModel"
             type="com.trikot.viewmodels.sample.viewmodels.ButtonListItemViewModel" />
-
-        <variable
-            name="lifecycleOwnerWrapper"
-            type="com.mirego.trikot.viewmodels.LifecycleOwnerWrapper" />
     </data>
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:lifecycleOwnerWrapper="@{lifecycleOwnerWrapper}"
         core:view_model="@{viewModel}">
```

Comment:
That's weird, some xml use `core:view_model` while others use `app:view_model`. 🤔  I think we should uniform this (in another PR maybe).